### PR TITLE
Sk sessions cluster

### PR DESCRIPTION
**Pull request by ShaneKilkelly**

### CODE REVIEW

File: app/coffee/Features/User/UserSessionsManager.coffee [hunk 1]
```diff
@@ -66,7 +68,8 @@ module.exports = UserSessionsManager =
 			if sessionKeys.length == 0
 				logger.log {user_id: user._id}, "no other sessions found, returning"
 				return callback(null, [])
-			rclient.mget sessionKeys, (err, sessions) ->
+
+			Async.map sessionKeys, ((k, cb) -> rclient.get(k, cb)), (err, sessions) ->
```
Comment (jpallen) [inline app/coffee/Features/User/UserSessionsManager.coffee]: I believe this all happens in a parallel. Perhaps make it mapSeries just to be safe?

File: config/settings.defaults.coffee [hunk 2]
```diff
@@ -107,8 +117,8 @@ module.exports = settings =
 		# references:
 		# 	url: "http://localhost:3040"
 		notifications:
-			url: "http://localhost:3042" 
-
```
Comment (jpallen) [inline config/settings.defaults.coffee]: I think your editor trims whitespace on exit, which makes commits more noisey. Consider turning it off? (Don't worry for this commit) Or we should do a single pass to clean up whitespace and then add a linting rule or something.

File: app/coffee/Features/User/UserSessionsManager.coffee [hunk 3]
```diff
@@ -3,13 +3,15 @@ redis = require('redis-sharelatex')
 logger = require("logger-sharelatex")
 Async = require('async')
 _ = require('underscore')
+UserSessionsRedis = require('./UserSessionsRedis')
 
-rclient = redis.createClient(Settings.redis.web)
+# rclient = redis.createClient(Settings.redis.web)
```
Comment (jpallen) [inline app/coffee/Features/User/UserSessionsManager.coffee]: clean up

File: app/coffee/Features/User/UserSessionsManager.coffee (hunk 1 above)
Comment (ShaneKilkelly) [inline app/coffee/Features/User/UserSessionsManager.coffee]: Will do

**Pull request merged**